src/hotspot/share/jfr/recorder/stacktrace/jfrAsyncStackTrace.cpp

```diff
@@ -67,7 +65,7 @@ bool JfrAsyncStackTrace::record_async(JavaThread* jt, const frame& frame) {
   u4 count = 0;
   _reached_root = true;
 
-  JfrVframeStream vfs(jt, frame, false, true);
+  JfrVframeStream vfs(jt, frame, false, true, false);
```

Comment:
Does this mean we are not going to get frames from virtual threads/continuations?

Reply:
Only the ones that are currently on the stack, as far as I understand.

But this change is required, as walking continuations in a signal causes crashes.

Reply:
Ok. This is benign then. Since the carrier thread is being executed the virtual thread frames (if any is mounted) will be on stack.